controllers/storagecluster/cephfilesystem.go

```diff
@@ -121,11 +122,86 @@ func (obj *ocsCephFilesystems) ensureCreated(r *StorageClusterReconciler, instan
 				return reconcile.Result{}, err
 			}
 		}
+		// create default csi subvolumegroup for the filesystem
+		// skip for the ocs provider mode
+		if !instance.Spec.AllowRemoteStorageConsumers {
```

Comment:
@Madhu-1 please confirm this

Reply:
@nb-ohad Can you pls confirm?

Reply:
If the intention here is to skip the default subvolumegropup for provider mode then it seems correct as we create a subvolumegropup per storageclassclaim

Reply:
Thanks

Reply:
Yes you thats correct we need to skip it for provider mode